modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/MockedStructuresTest.java

```diff
@@ -275,6 +285,58 @@ private void mockMetastore() throws Exception {
         when(cursorMocked.iterator()).thenReturn(itMock);
     }
 
+    @Test
+    public void testCreateZone() {
```

Comment:
does it make sense to move all zone-related methods to a separate class?

Reply:
Done, thanks.